### PR TITLE
BZ1915234 Button says *Save* instead of *Update*.

### PR DESCRIPTION
**Pull request by stoobie**
https://bugzilla.redhat.com/show_bug.cgi?id=1915234
enterprise 4.8
enterprise 4.7
enterprise 4.6

### CODE REVIEW

**Review by kalexand-rh — COMMENTED** [review]
This change LGTM!

File: modules/update-upgrading-web.adoc [hunk 1]
```diff
@@ -53,8 +53,8 @@ endif::within[]
 ifdef::between[]
 the highest available version
 endif::between[]
-and click *Update*.
-The *Update Status* changes to `Update to <product-version> in progress`, and you can review the progress of the cluster update by watching the progress bars for the Operators and nodes.
+and click *Save*.
+*Update Status* changes to *Update to <product-version> in progress*, and you can review the progress of the cluster update by watching the progress bars for the Operators and nodes.
```
[the remark this answers is not in the document]
Comment (mgold1234) [reply modules/update-upgrading-web.adoc:57]: *Select Channel- appears with small 'c' letter.

[the remark this answers is not in the document]
Comment (stoobie) [reply modules/update-upgrading-web.adoc]: @mgold1234 Fixed. Please check and approve the changes.

**Comment by kalexand-rh** [comment]
@mgold1234, PTAL?

**Comment by mgold1234** [comment]
image
please change

**Comment by bergerhoffer** [comment]
@stoobie The cherry pick to 4.6 failed due to a conflict. Can you please submit a manual cherry pick for it?

**Pull request merged**